susemanager/bin/mgr-setup

```diff
@@ -437,6 +437,8 @@ done
 
 do_setup
 
+/usr/sbin/spacewalk-service stop
```

Comment:
The start and the matching stop are only needed for internal DB. Could you please merge the two following ifs and have them inside the if?

Reply:
Done. I saw those two conditions, but was too eager to see CI pass 😄